src/evm/process-sponsor-wallet.ts

```diff
@@ -62,7 +62,7 @@ export const processSponsorWallet = async (
               nonce,
             }
           ),
-      { timeoutMs: TIMEOUT_MS, retries: RETRIES }
```

Comment:
@amarthadan @acenolaza  I don't understand why adding retries here is causing `transactions-count.test.ts` to fail :confused:

Reply:
I think the problem is that I used the `mockRejectedValueOnce` function and we are now callling it more than once causing the test to timeout. You could try using `mockRejectedValue` or `mockImplementation`